tools/accuracy_checker/accuracy_checker/metrics/classification.py

```diff
@@ -262,3 +266,26 @@ def reset(self):
         self.tn = 0
         self.fp = 0
         self.fn = 0
+
+class RocAucScore(PerImageEvaluationMetric):
+    __provider__ = 'roc_auc_score'
+    annotation_types = (ClassificationAnnotation, TextClassificationAnnotation)
+    prediction_types = (ClassificationPrediction, ArgMaxClassificationPrediction)
+
+    def configure(self):
+        self.reset()
```

Comment:
please add check that  roc_auc_score is not None

Reply:
and please add new metric to the readme

Reply:
Fixed